### PR TITLE
Update syslog

### PR DESCRIPTION
**Pull request by bkochauri-memphis**
## How Has This Been Tested?

Please describe the tests you ran to verify your changes and note any relevant details for your test configuration.
- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] I have performed a self-review of my code
- [ ] I have made corresponding changes to the knowledge base (if needed)
- [ ] My changes generate no new warnings
- [ ] I have verified that the specification is met and all functionalities are working as expected

## Reviewer Score - 0-100%
- [ ] **Tests have been done and meaningfully described - (25%)**
   - Ensure all necessary tests are conducted, covering the critical paths and edge cases.
   - Descriptions should be clear, documenting the test intent, steps, and outcomes for reproducibility and transparency.
     
- [ ] **Meeting Task Specifications (25%)**
   - This includes security, observability, and cost-efficiency aspects.
   - Ensure that the task requirements are fully met and that the implementation aligns with the specifications provided.
     
- [ ] **Writing Readable Code (20%)**
   - Write code that is easy to read and understand.
   - Use clear and meaningful variable names, and include comments where necessary.

- [ ] **Attention to the folder structures in automation tools and repositories (20%)**
   - Make sure the folder hierarchy and the names are clear and meaningful.

- [ ] **Attention to Edge Cases (10%)**
   - Identify and handle edge cases that may not be immediately obvious.
   - Demonstrate thorough testing and consideration of potential issues.

### CODE REVIEW

File: version.conf [hunk 1]
```diff
@@ -1 +1 @@
-0.5.2
\ No newline at end of file
+0.5.3
```
Comment (valeraBr) [inline version.conf]: Raise to 0.6.0 please @bkochauri-memphis

**Review by valeraBr — APPROVED** [review]

**Review by valeraBr — APPROVED** [review]

File: version.conf (hunk 1 above)
Comment (bkochauri-memphis) [reply version.conf]: Done
